File: tests/fixtures/repos/py-sample/telemetry.py
from dataclasses import dataclass
import time


@dataclass
class TelemetryEvent:
    name: str
    payload: dict
    timestamp: float


def log_event(name: str, payload: dict) -> None:
    event = TelemetryEvent(name=name, payload=payload, timestamp=time.time())
    _ = event  # would emit to a sink in a real system

File: apps/api/test/legacy/fixtures/repos/ts-sample/src/telemetry.ts
export interface TelemetryEvent {
  name: string
  payload: Record<string, unknown>
  timestamp: number
}

export function logEvent(name: string, payload: Record<string, unknown>): void {
  const event: TelemetryEvent = { name, payload, timestamp: Date.now() }
  // In a real system this would emit to a sink.
  void event
}
